pkgs/development/interpreters/ruby/default.nix

```diff
@@ -34,6 +35,7 @@ let
       , libyaml, yamlSupport ? true
       , libffi, fiddleSupport ? true
       , jemalloc, jemallocSupport ? false
+      , linuxPackages, systemtap ? linuxPackages.systemtap, libsystemtap, dtraceSupport ? false
```

Comment:
`systemtap` in theory depends on the kernel version. In practice it (currently) doesn't for the `dtrace` utility.
This may get less messy in the future.